restapi/restapiserver.go

```diff
@@ -0,0 +1,8 @@
+// Package restapi is where all restapi go
+package restapi
```

Comment:
I think this file should be part of other PR.

Reply:
OK, will remove it later